app/assets/javascripts/application.js

```diff
@@ -16,6 +16,11 @@
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require controllers/deployment/wizard-module
```

Comment:
There was a dependency order problem with just requiring all the controller tree.
There is still no reason to review it, it is still to early :) . Thanks.